test_app/run_product_tests.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd /Users/macbookairm432g/work/railsapps/swiftrailsviews/test_app
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)